System/Console/ANSI/Codes.hs

```diff
@@ -60,13 +60,18 @@ module System.Console.ANSI.Codes
   , setTitleCode
 
     -- * Utilities
-  , colorToCode, csi, sgrToCode
+  , colorToCode
+  , color8CodeToXterm256
```

Comment:
I am wondering if this module needs to export `color8CodeToXterm256`, `xterm256ColorToCode` or `closestXterm256Color`, if module `System.Console.ANSI.Color` is to be exposed.

Reply:
yes, these can be removed